### PR TITLE
Metadata for full range mkv

### LINKED ISSUE #87: Add metadata for full range recordings
Since vrecord now has a playback view option for full range capture, this should be reflected in metadata of the resulting video files. Metadata should specify whether a video file should be interpreted as broadcast range or full range.

### PR DESCRIPTION
**Pull request by savcampbell**
mkv captures can now be interpreted as Broadcast Range or Full Range
depending on selected playback view. Fixes #87

### CODE REVIEW

**Comment by dericed** [comment]
I wonder if these should be two separate options. @kellyhaydon do you want to record a full range file while you use a full range waveform viewer or just record a broadcast range file while you use a full range waveform viewer

**Comment by kellyhaydon** [comment]
It would depend on the format, so having options makes sense.

**Comment by retokromer** [comment]
What is the status of this? Is @savcampbell still working on vrecord?

**Comment by retokromer** [comment]
Ping! Can this be merged?

**Comment by dericed** [comment]
Adding self to assignment.

File: vrecord [hunk 1]
```diff
@@ -1096,17 +1096,23 @@ if [[ "${standard_choice}" = "PAL" ]] ; then
...
     recordingfilter="[0:v:0]setfield=bff,setsar=40/27,setdar=4/3[vid1]"
 fi
 
+if [[ "${playbackview_choice}" = "Broadcast Range Visual" ]] ; then
+    middleoptions+=(-color_range mpeg)
+elif [[ "${playbackview_choice}" = "Full Range Visual" ]] ; then
+    middleoptions+=(-color_range jpeg)
+else
+    middleoptions+=(-color_range mpeg)
+fi
```
Comment (retokromer) [inline vrecord]: Are these tests not equivalent to this one?
```
if [[ "${playbackview_choice}" = "Full Range Visual" ]] ; then
    middleoptions+=(-color_range jpeg)
else
    middleoptions+=(-color_range mpeg)
fi
```

**Comment by retokromer** [comment]
@dericed I simplified the test (please check if I am not missing something), because @savcampbell seems no longer to be working on vrecord.

**Comment by retokromer** [comment]
ping

**Comment by dericed** [comment]
Is this tested? From a skim it seems that in some conditions that -color_range could be declared twice.

**Comment by retokromer** [comment]
I am not sure I understand you carefully. The `-color_range` value is set only in https://github.com/savcampbell/vrecord/blob/0c94cb16340e61d54e946e4d6068e6dab79bd325/vrecord#L1108-L1112 which in the current version depends on one single test.

**Comment by dericed** [comment]
ah ok i misread the patch.

**Pull request merged**